app/routes/dev.tsx

```diff
@@ -56,6 +57,30 @@ export const action = async ({ request }: ActionArgs) => {
         result: 'ok',
         message: 'DNS record creation requested',
       });
+    case 'update-dns-record-request':
+      await updateDnsRequest({
+        id: 62,
+        type: 'A',
+        name: 'osd700-a2.user1.starchart.com',
```

Comment:
I think that we should only require passing the first part of the `name`, and not the base domain, which we can get from the user (see #298).